src/applications/mhv/medications/components/shared/ExtraDetails.jsx

```diff
@@ -65,29 +65,39 @@ const ExtraDetails = rx => {
           <va-link href="/" text="Compose a message" />
         </div>
       )}
-      {refillStatus === 'transferred' && (
+      {dispStatus === 'Transferred' && (
         <div>
           <p className="vads-u-margin-y--0">
             To manage this prescription, go to our My VA Health portal.
           </p>
           <va-link href="/" text="Go to your prescription in My VA Health" />
         </div>
       )}
-      {refillStatus === 'non-va' && (
+      {dispStatus === 'Active: Non-VA' && (
         <div>
           <p className="vads-u-margin-y--0">
             This isn’t a prescription that you filled through a VA pharmacy. You
             can’t manage this medication in this online tool.
           </p>
         </div>
       )}
+      {dispStatus === 'Active: On Hold' && (
+        <div className="no-print">
+          <p className="vads-u-margin-y--0">
+            We put a hold on this prescription. If you need it now, call your VA
+            pharmacy at <va-telephone contact="3538675309" /> (
```

Comment:
I believe that this pharmacy number is a fake number. The number should come from the `cmopDivisionPhone` field, instead of it being hard coded. You could use this shared component `src/applications/mhv/medications/components/shared/CallPharmacyPhone.jsx`. It expects the phone number and if none is provided it just returns a period.

Reply:
changed